From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Fri, 10 Feb 2017 13:16:26 +0100
Subject: Link against GIO_LIBS explicitly

to avoid

libtool: link: gcc -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -g -O2 -fdebug-prefix-map=/build/gtk-vnc-0.6.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wl,-z -Wl,relro
 -Wl,-z -Wl,now -o .libs/vncconnectiontest vncconnectiontest-vncconnectiontest.o  ./.libs/libgvnc-1.0.so -lz -pthread
/usr/bin/ld: vncconnectiontest-vncconnectiontest.o: undefined reference to symbol 'g_io_stream_get_output_stream'
//usr/lib/x86_64-linux-gnu/libgio-2.0.so.0: error adding symbols: DSO missing from command line

Also make the use of *_CFLAGS and *_LIBS match.
---
 src/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index f7c1d9d..8bc9085 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -317,8 +317,8 @@ BUILT_SOURCES += $(MARSHAL_FILES) $(ENUM_FILES)
 CLEANFILES = $(MARSHAL_FILES) $(ENUM_FILES)
 
 vncconnectiontest_SOURCES = vncconnectiontest.c
-vncconnectiontest_CFLAGS = $(GOBJECT_CFLAGS)
-vncconnectiontest_LDADD = libgvnc-1.0.la
+vncconnectiontest_CFLAGS = $(GOBJECT_CFLAGS) $(GIO_CLFAGS)
+vncconnectiontest_LDADD = libgvnc-1.0.la $(GOBJECT_LIBS) $(GIO_LIBS)
 
 if WITH_PYTHON
 pyexec_LTLIBRARIES = gtkvnc.la
